less/index.less

```diff
@@ -66,111 +120,143 @@ h1, h2, h3, h4, h5 {
+    @media (max-width: 400px) {
+        background: spin(saturate(@headerbg, 10%), 20);
+    }
+    nav {
+        margin-top: 40px; 
+        width: 610px;
+        display: flex;
+        justify-content: space-around;
+        align-items: center;
+        a {
+            text-decoration: none;
+            font-weight: bold;
+            color: @fontcolor;
+        }
+        a:hover {
+            text-decoration: underline;
+            color: white;
+        }
+    } 
+    .logo {
     align-self: flex-end;
-    margin: 28px 0 0 91px;
+    margin: 28px 5px 0 91px;
+    }
 }
-
 /* Call To Action (CTA) Styles */
 
 .cta {
     display: flex;
     margin-top: 72px;
     justify-content: flex-end;
-}
-
-.cta .cta-text {
-    font-size: 72px;
-    text-align: center;
-    letter-spacing: 5px;
-    display: flex;
-    justify-content: center;
-    flex-flow: column;
-    align-items: center;
-    margin: 0 82px 0 0;
-}
-
-.cta .cta-text button {
-    border: 1px double black;
-    font-size: 16px;
-    background: white;
-    width: 170px;
-    height: 32px;
-    margin-top: 10px;
-    cursor: pointer;
-}
-
-.cta .cta-text button:hover {
-    background: black;
-    color: white;
+    background: @ctabg;
+    @media (max-width: 800px) {
+        background: spin(saturate(@ctabg, 10%), 10);
+    }
+    @media (max-width: 400px) {
+        background: spin(saturate(@ctabg, 10%), 20);
+    }
+    .cta-text {
+        text-align: center;
+        letter-spacing: 5px;
+        display: flex;
+        justify-content: center;
+        flex-flow: column;
+        align-items: center;
+        margin: 0 82px 0 0;
+        .fontsizeConverter(72);
+        button {
+            border: 1px double black;
+            background: white;
+            width: 170px;
+            height: 32px;
+            margin-top: 10px;
+            cursor: pointer;
+            .fontsizeConverter(16);
+        }
+        button:hover {
+            background: black;
+            color: white;
+        }
+    }
 }
 
 /* Main Content Styles */
 
 .main-content {
-   border-top: 2px solid black; 
-   border-bottom: 2px solid black;
-   margin: 75px 0 35px 0;
-   padding: 35px 0;
-   font-size: 16px;
-   line-height: 18px;
-}
-
-.main-content .top-content {
-    display: flex;
-    justify-content: space-around;
-}
-
-.main-content .middle-img {
-    display: block;
-    margin: 30px auto;
-}
-
-.main-content .bottom-content {
-    display: flex;
-    justify-content: space-around;
-}
-
-.main-content .bottom-content p {
-    padding-right: 20px;
+    border-top: 2px solid black; 
+    border-bottom: 2px solid black;
+    margin-top: 75px;
+    padding: 35px 0;
+    line-height: 18px;
+    background: @contentbg;
+    color: white;
+    .fontsizeConverter(16);
+    @media (max-width: 800px) {
+        background: spin(saturate(@contentbg, 10%), 10);
+    }
+    @media (max-width: 400px) {
+        background: spin(saturate(@contentbg, 10%), 20);
+    }
+    .top-content {
```

Comment:
Here's a ton of food for thought:

So first, you could use a single mixin that has `display: flex` and `justify-content: space-around` that would then be applied to both `.top-content` and `.bottom-content`. But let's take this further, theoretically. I'll share one interesting idea another student employed:

>`//.flex-mix(@dir: column; @align: center); || .flex-mix(@just: center;) || etc`
>`.flex-mix(@just: flex-start; @align: stretch; @dir: row; @wrap: nowrap) {`

>>  `display: flex;`
>  `justify-content: @just;`
>  `align-items: @align;`
>  `flex-flow: @dir @wrap;`

>`}`

Sorry if it comes out weird, I'm having trouble getting github to style it correctly. Anyway, this is kind of a neat idea. I'm showing you this to get you thinking about how you might employ some parametric mixins with some default settings as well. (So with this, you never have to give all or any values into the mixin.) I'm not saying this is the best application, but it shows you some possibilities.